### ISSUE TITLE
incorrect lemma from lemmatizer 

### ISSUE THREAD

**Issue by oltip** [explosion/spaCy#3700]
## How to reproduce the behavior

I am not sure it's a bug, but just wanted to let you know that the lemma of the following words: 'car', 'carriers', 'scar',  (SIM) card, 
is always car. Is it correct from the lemmatization point of views? 

## Your Environment 

* Operating System: windows 
* Python Version Used: 3.7 
* spaCy Version Used: 2.1.1 
* Environment Information: conda

**Comment by bjascob** [comment]
Spacy has a number of lemmatization issues.  I have a list of about 400 words that lemmatize incorrectly posted in issue #3665.  The current release also has an issue where the lemma can randomly vary.  This is fixed in PR #3646.  That PR is merged but it hasn't made a release yet.

**Comment by honnibal** [comment]
I agree there are outstanding lemmatization issues, but does `scar` really lemmatize to `car`?? I can't see why that would happen, and can't immediately reproduce it:

```
>>> import spacy
>>> nlp = spacy.load('en_core_web_sm')
>>> doc = nlp(u'He has a scar.')
>>> doc[-2].lemma_
'scar'```

@oltip I'd love to get to the bottom of what you've observed, if you can provide more details. Could you try again with v2.1.3? We did fix a bug in v2.1.1 that might be causing the behaviour you're seeing.
